docs/excel/custom-functions-architecture.md

```diff
@@ -0,0 +1,42 @@
+---
+ms.date: 03/06/2019
+description: Learn about Excel custom functions' runtime. 
+title: Custom functions architecture (preview)
+localization_priority: Priority
+---
+# Custom functions architecture
+
+Custom function use a new JavaScript runtime that differs from the browser-based JavaScript engine which powers most other parts of your add-in.
+
+## Custom functions runtime
+
+The custom functions runtime prioritizes executing calculations, so functions run run smoothly when Excel recalculates. The custom functions runtime allows access to APIs which make requesting external data and exchanging data over a persistent connection over a server possible.
```

Comment:
I'm not clear on the second sentence. Is it referring to APIs in the JS library that enable requesting external data?

Reply:
Yeah - because it also allows you to get data from your own server via websockets it seemed weird to just say "it can also do web things!" but happy to rephrase.